kolibri/plugins/coach/assets/src/views/plan/LessonCreationPage/index.vue

```diff
@@ -51,6 +51,15 @@
         return this.$route.params.classId;
       },
     },
+    async created() {
+      const initClassInfoPromise = this.$store.dispatch('initClassInfo', this.classId);
+      const getFacilitiesPromise =
+        this.$store.getters.isSuperuser && this.$store.state.core.facilities.length === 0
+          ? this.$store.dispatch('getFacilities').catch(() => {})
+          : Promise.resolve();
+
+      await Promise.all([initClassInfoPromise, getFacilitiesPromise]);
```

Comment:
I think that we can remove async and await from `created` (it's a different situation from handlers where we needed to wait because there were other promises that needed to be run after we are sure these are finished). Responsivity should take care of everything, ideally ;)